docs/importing_autokey_key_handles.md

```diff
@@ -0,0 +1,53 @@
+# Importing Autokey Key Handles Guidance
+
+If you have any existing [Autokey Key Handles](https://cloud.google.com/kms/docs/resource-hierarchy#key_handles) previously created manually or using [terraform-google-autokey](https://registry.terraform.io/modules/GoogleCloudPlatform/autokey/google) module, it is possible to import them to [autokey submodule](../modules/autokey/README.md) Terraform state by following the steps below.
+
+**Note:** You don't need to import the existing [Autokey configuration](https://cloud.google.com/kms/docs/enable-autokey#enable-autokey-folder) resource. The [autokey submodule](../modules/autokey/README.md) apply process will handle that automatically.
+
+## Importing from terraform-google-autokey considerations
+1. **WARNING:** [terraform-google-autokey](https://registry.terraform.io/modules/GoogleCloudPlatform/autokey/google) module can be used to create your Autokey folder, Autokey KMS project, Autokey resource project and additional resources (e.g: a Cloud Storage Bucket configured with Autokey), so **DO NOT RUN** a `terraform destroy` for the module, even after the Key Handle import, unless you just used the module for Autokey configuration and the Key Handle creations only.
+1. If you are migrating from [terraform-google-autokey](https://registry.terraform.io/modules/GoogleCloudPlatform/autokey/google) module, you should copy the terraform outputs provided by the module and jump directly to step 4.
+
+## Steps
+
+1. Identify your existing Autokey folder number used to configure [Autokey configuration](https://cloud.google.com/kms/docs/enable-autokey#enable-autokey-folder).
+    1. This value will be referenced as: `AUTOKEY-FOLDER-NUMBER`
+1. Identify your existing project ID used to [configure Cloud KMS resources created by Autokey](https://cloud.google.com/kms/docs/enable-autokey#set-up-key-project).
+    1. This value will be referenced as: `AUTOKEY-KMS-PROJECT-ID`
+1. Indentify your existing [Autokey Key Handle](https://cloud.google.com/kms/docs/resource-hierarchy#key_handles) configuration. It's composed by the following:
+    1. Autokey Resource Project ID; (e.g: A Cloud Storage Bucket project ID configured with Autokey);
+        1. This value will be referenced as: `AUTOKEY-RESOURCE-PROJECT-ID`;
+    1. Key Handle location; (e.g: A Cloud Storage Bucket location configured with Autokey);
+        1. This value will be referenced as: `AUTOKEY-LOCATION`;
+    1. Key Handle name; (e.g: The Key Handle name configured for Cloud Storage Bucket);
+        1. This value will be referenced as: `AUTOKEY-KEYHANDLE-NAME`;
```

Comment:
Could a example be added with:
- Terraform commands on how to get these values from an existing state 
- `gcloud`  commands to get these values from GCP

Reply:
+1, if we can provide a bash script that would be helpful.

Reply:
or if you want to try creating some simple go code using the templating library for the KH's